build.properties

```diff
@@ -8,12 +8,12 @@ metacat.version=3.0.0
 metacat.releaseCandidate=
 
 # Tomcat dev deployment directory
-build.tomcat.dir=/usr/share/tomcat8
+build.tomcat.dir=/usr/share/tomcat9
 #build.tomcat.dir=/Users/leinfelder/tools/apache-tomcat-6
 # install ant target uses this to determine where to drop 
 # the war file in an installation.  Test classes will use
 # this value to find metacat.properties
-app.deploy.dir=/opt/local/share/java/tomcat8/webapps
+app.deploy.dir=/opt/local/share/java/tomcat9/webapps
```

Comment:
instead of having to continue editing this each time we upgrade tomcat, would it be feasible to just use a default directory name of `tomcat` instead of `tomcat9`? The user can then decide either to use a symlink, or to change this property. 

My motivation: I didn't want to change this property because git would then always show it as modified (unless I add it to .gitignore, which could lead to other problems). I therefore added a symlink, which currently looks like this - not a great experience 🤣:
```console
$ la /opt/local/share/java/
total 8
drwxr-xr-x  4 root  wheel  -  128B Sep 21 20:07 ./
drwxr-xr-x  3 root  wheel  -   96B Feb  8  2023 ../
-rw-r--r--  1 root  wheel  -  220B Sep 21 20:07 README.md
lrwxr-xr-x  1 root  wheel  -   34B Sep 21 20:04 tomcat8 -> /opt/homebrew/opt/tomcat@9/libexec
```

In case you're curious, here's what I wrote in README.md (I wrote it as a reminder after my build broke when I upgraded to tomcat 9):

```console
$ cat /opt/local/share/java/README.md

# SHAMELESS HACK!

tomcat8 points to tomcat9 because tomcat8 is used in metacat's ant build.properties:

`app.deploy.dir=/opt/local/share/java/tomcat8/webapps`

...and I don't want to keep modifying it
```

Reply:
There is no easy ways to accommodate different settings. The purpose of the build.properties files is to give users a way to customize their settings. So changing on this file is every common, just like the values.yam on k8s.

Reply:
I understand that. If it is common to edit this file, that also means we can choose any default value we want.

I therefore think it makes sense to choose a default value that is not tied to a particular version of tomcat. This will reduce maintenance overhead and potential mistakes.

Reply:
That sounds good. I will remove the version there.